aws/terraform/infra/README.md

```diff
@@ -0,0 +1,29 @@
+# Infra
+
+Stands up the base infrastructure required to deploy a Metaflow stack.
+
+Mostly stands up and configures the VPC.
+
+## AWS Resources
+
+### IAM
```

Comment:
Minor nit (here and elsewhere and very nit-picky, so feel free to ignore) - For consistency with the service name usage in CFN, can we use the proper conventional names - Amazon IAM, Amazon S3, AWS Batch etc.

Reply:
Yeah sure thing, I'll dive through the docs and add the AWS prefix for the Amazon products.